### DISCUSSION TITLE
ImageMagick is cloud base or on-premise?

### DISCUSSION THREAD

**Discussion by aslotvitskaya** [ImageMagick/ImageMagick#5374] [Help] [answered]
ImageMagick is cloud base or on-premise?

**Comment by dlemstra** [comment]
ImageMagick is software and not a service. What are you trying to ask us?

**Reply by aslotvitskaya** [reply]
ImageMagick is a program that is installed on our PC using a distribution kit for installation on our server? Or ImageMagick is deployed on a remote server?

**Comment by snibgo** [comment]
ImageMagick can be installed on many types of computers. It can be installed on PCs, and can be installed on servers.

**Reply by aslotvitskaya** [reply]
If I want to install it on PC, it will be work without an internet connection?

**Comment by snibgo** [comment, marked as answer]
When it is installed, ImageMagick does not need an internet connection.
